tools/ark-cli/src/lib/executeQuery.ts

```diff
@@ -207,6 +215,85 @@ async function executeQueryWithFormat(options: QueryOptions): Promise<void> {
   }
 }
 
+async function watchEventsLive(queryName: string): Promise<void> {
+  const seenEvents = new Set<string>();
+
+  const pollEvents = async () => {
+    try {
+      const {stdout} = await execa('kubectl', [
+        'get',
+        'events',
+        '--field-selector',
+        `involvedObject.name=${queryName}`,
+        '-n',
+        'default',
+        '-o',
+        'json',
+      ]);
+
+      const eventsData = JSON.parse(stdout);
+      for (const event of eventsData.items || []) {
+        const eventId = event.metadata?.uid;
+
+        if (eventId && !seenEvents.has(eventId)) {
+          seenEvents.add(eventId);
+
+          const annotations = event.metadata?.annotations || {};
+          const eventData = annotations['ark.mckinsey.com/event-data'];
```

Comment:
can probably pull to a const. one day a node sdk client lib will be useful but who knows when we'll get to the time